frontend/src/lib/lemon-ui/LemonWidget/LemonWidget.tsx

```diff
@@ -1,47 +1,26 @@
-import { useState } from 'react'
 import { LemonButton } from '../LemonButton'
-import { IconClose, IconUnfoldLess, IconUnfoldMore } from '../icons'
+import { IconClose } from '../icons'
 import './LemonWidget.scss'
 import clsx from 'clsx'
 
 export interface LemonWidgetProps {
     title: string
-    collapsible?: boolean
     onClose?: () => void
     actions?: React.ReactNode
-    children: React.ReactChild
+    children: React.ReactNode
+    className?: string
```

Comment:
Still not sure if we should be letting people add this tbh

Reply:
Even inside our UI components? I was basing that off a lot of our other existing components which allow similar customisation